application/tests/unit/common/models/EmailTest.php

```diff
@@ -171,7 +171,8 @@ public function testSend()
 
         $this->assertEquals($initialEmailQueueCount + 1, Email::find()->count(), 'emails in db did not increase by one after saving email');
 
-        $email->send();
+        $n = $email->send();
+        $this->assertEquals(1, $n, 'message not sent');
```

Comment:
I'm curious about the reason behind this change... it looks like this `send()` method was changed from throwing an exception to simply returning a value, but where it's used was updated to take that value and throw an exception if it indicates that the message wasn't sent. What was gained by changing from an exception to a return value?

Reply:
It's a slight simplification, but my intent was to remove what I perceived to be a code smell of returning a value in an exception. I'm pretty sure I wrote it this way, but didn't like reading it months later. A delayed send isn't really an exception condition in my current opinion.